Source/WebCore/loader/ImageLoader.cpp

```diff
@@ -359,7 +359,7 @@ CachedResourceHandle<CachedImage> ImageLoader::protectedImage() const
 
 void ImageLoader::updateFromElementIgnoringPreviousErrorToSameValue()
 {
-    if (!m_image || !m_image->allowsCaching() || !m_failedLoadURL.isEmpty() || element().document().activeServiceWorker()) {
+    if (!m_image || !m_image->allowsCaching() || !m_failedLoadURL.isEmpty() || protectedDocument()->activeServiceWorker()) {
```

Comment:
This is silly. We should just move the definition of `ScriptExecutionContext::activeServiceWorker` to the header file.

Reply:
Done.